test_data/asm/batching/labels.asm

```diff
@@ -1,17 +1,21 @@
-degree 256;
-reg pc[@pc];
-reg X[<=];
-reg A;
+machine Main {
+    degree 256;
+    reg pc[@pc];
+    reg X[<=];
+    reg A;
 
-instr loop { pc' = pc }
+    instr loop { pc' = pc }
 
-label_with_next::
- A <=X= 1;
+    operation main {
```

Comment:
What is the difference between `operation main` and `program`?

Reply:
`program` does not exist in the parser anymore. It is the rom generated during compilation by joining all operations.